hcn/hcnpolicy.go

```diff
@@ -127,11 +127,12 @@ type QosPolicySetting struct {
 
 // OutboundNatPolicySetting sets outbound Network Address Translation on an Endpoint.
 type OutboundNatPolicySetting struct {
-	VirtualIP  string   `json:",omitempty"`
-	Exceptions []string `json:",omitempty"`
+	VirtualIP    string   `json:",omitempty"`
+	Exceptions   []string `json:",omitempty"`
+	Destinations []string `json:",omitempty"`
 }
 
-// SDNRoutePolicySetting sets SDN Route on an Endpoint.
+// Sdnroutepolicysetting sets SDN Route on an Endpoint.
```

Comment:
This was correct before. The comment must match the struct name

Reply:
No clue why that got changed.